packages/openactive-integration-tests/documentation/generator.js

```diff
@@ -77,6 +77,15 @@ const INDEX_TESTS_IMPLEMENTED_JSON_FILE = path.join(FEATURES_ROOT, 'tests-implem
  * }} FeatureMetadataItem
  */
 
+const FEATURES_NOT_REQUIRED_FOR_DEFAULT_JSON = new Set([
+  'customer-requested-cancellation-always-allowed',
```

Comment:
Is the reason this is not in the default because customer cancellation are not always allowed? If so it's a weird feature but ok 🤷🏽